fineract-provider/build.gradle

```diff
@@ -436,7 +436,8 @@ modernizer {
 
     ignorePackages = []
     ignoreClassNamePatterns = [
-        '.*AbstractPersistableCustom'
+        '.*AbstractPersistableCustom',
+        '.*EntityTables'
```

Comment:
Errorprone wants all fields in Enumeration to Immutable, hence Integer[] is require to be ImmutableList<Integer> and modernizer wants all ImmutableList.of to be List.of which Error prone doesn't consider as Immutable, hence had to add EntityTables to this list.

Reply:
understood. Thanks for the explanation.